CMakeLists.txt

```diff
@@ -0,0 +1,74 @@
+cmake_minimum_required(VERSION 3.16)
+project(isaac_ros2_control CXX)
```

Comment:
Whats the `CXX` for?

Reply:
Just curious why its required. I have not been giving my projects names that include `CXX`. Is this a style thing?

Reply:
Yeah, I think it's a style thing, it's not required at all